### PR TITLE
feat: added alert component

### PR DESCRIPTION
**Pull request by vtrbo**

### CODE REVIEW

File: components/Alert/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+	"name": "@ikun-ui/alert",
+	"version": "0.0.9-beta.2",
```
Comment (baiwusanyu-c) [inline components/Alert/package.json:3]:
```suggestion
	"version": "0.0.9-beta.3",
```

File: components/Alert/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	"type": "module",
+	"main": "dist/index.js",
```
Comment (baiwusanyu-c) [inline components/Alert/package.json:5]: sync pr #174

File: components/Alert/__test__/alert.spec.ts [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+	});
+
+	test('slots: closeIcon', async () => {
+		const instance = new KAlertClose({
+			target: host
+		});
+		expect(instance).toBeTruthy();
+		expect(host.querySelector('#alert-close')?.textContent).toBe('this is close slot');
+		expect(host.innerHTML).matchSnapshot();
+	});
```
Comment (baiwusanyu-c) [inline components/Alert/__test__/alert.spec.ts:183]: Add test for closing event

File: preset/src/shortcuts/alert.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+	const descriptionFontSize = '12px';
+	const descriptionMt = '8px';
+
+	// close
+	const closeTop = '10px';
+	const closeTopNoTitle = '8px';
+	const closeFontSize = '12px';
+
+	return {
+		// alert
```
Comment (baiwusanyu-c) [inline preset/src/shortcuts/alert.ts:47]: I prefer shortcut to use class objects to exhaustively list atomic class names, too abstract and not conducive to maintenance.
wdyt？

**Review by baiwusanyu-c — CHANGES_REQUESTED** [review]

**Comment by baiwusanyu-c** [comment]
image
Doesn't seem to be vertically centered

File: preset/src/shortcuts/alert.ts (hunk 4 above)
Comment (vtrbo) [reply preset/src/shortcuts/alert.ts:47]: > I prefer shortcut to use class objects to exhaustively list atomic class names, too abstract and not conducive to maintenance. wdyt？

This is just an attempt to reduce a lot of duplicate code. However, there are indeed difficult maintenance problems.

**Comment by vtrbo** [comment]
> image Doesn't seem to be vertically centered

I fixed it

**Review by baiwusanyu-c — APPROVED** [review]

**Pull request merged**